client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -89,7 +90,7 @@ const ScreenMain = ( { onSelect, onContinueClick }: Props ) => {
 			</div>
 			<div className="screen-container__footer">
 				<Button className="pattern-assembler__button" onClick={ onContinueClick } primary>
-					{ translate( 'Continue' ) }
+					{ shouldUnlockGlobalStyles ? translate( 'Unlock this design' ) : translate( 'Continue' ) }
```

Comment:
IMO, the action button should match the one in the upsell modal for consistency.
![Screenshot 2023-03-06 at 12 05 06 PM](https://user-images.githubusercontent.com/797888/223017166-cf4ccb8a-0d0e-4e35-85b6-2dd1909f93d1.png)

So either we change the button text to "Unlock this style" (a bit odd) or change the modal to accept "Unlock this design" as a component prop. What do you think?

Reply:
I changed the button text to "Unlock this style" 👍